src/AbstractSerializer.php

```diff
@@ -141,7 +143,7 @@ protected static function serializeHeaders(array $headers) : string
     /**
      * Filter a header name to wordcase
      */
-    protected static function filterHeader($header) : string
+    protected static function filterHeader(string $header) : string
```

Comment:
Adding native type-hint to non-private methods on a non-final class is a BC break due to mismatching signature.